cub/cub/device/device_adjacent_difference.cuh

```diff
@@ -282,6 +283,7 @@ public:
 
     return SubtractLeftCopy(d_temp_storage, temp_storage_bytes, d_input, d_output, num_items, difference_op, stream);
   }
+#endif
```

Comment:
We strife to not leave `#endif` without a matching comment, as it often is hard to see what branch this ends. I am pushing a comment